internal/config/config.go

```diff
@@ -65,6 +65,9 @@ type AppConfig struct {
 		// The WireGuard ListenPort
 		// Defaults to 51820
 		Port int `yaml:"port"`
+		// The maximum transmission unit
+		// Defaults to 1360
+		Mtu int `yaml:"mtu"`
```

Comment:
In go, initialisms [should be uppercase](https://github.com/golang/go/wiki/CodeReviewComments#initialisms)

> Words in names that are initialisms or acronyms (e.g. "URL" or "NATO") have a consistent case. For example, "URL" should appear as "URL" or "url" (as in "urlPony", or "URLPony"), never as "Url". As an example: ServeHTTP not ServeHttp. For identifiers with multiple initialized "words", use for example "xmlHTTPRequest" or "XMLHTTPRequest".

Hence `MTU` (as for example `CIDR` below).